### PR TITLE
rerun with preproccess

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR reruns the CellProfiler feature pipeline with adding in file paths features and X_Y center features as metadata.

### CODE REVIEW

File: 2.cellprofiler_ic_processing/scripts/perform_ic.py [hunk 1]
```diff
@@ -18,14 +18,13 @@
 import cp_utils as cp_utils
 import tqdm
 
-
 # ## Set paths
 
 # In[ ]:
```
Comment (jenna-tomkinson) [inline 2.cellprofiler_ic_processing/scripts/perform_ic.py:23]: Recommend making sure this file is run sequentially until the last code cell prior to nbconvert.

File: 1.pre_process_data/notebooks/1.generate_platemap.ipynb
Comment (jenna-tomkinson) [inline 1.pre_process_data/notebooks/1.generate_platemap.ipynb:1]: Note to make sure that this is nbconverted.

File: 1.pre_process_data/notebooks/0.fix_pathing.ipynb
Comment (jenna-tomkinson) [inline 1.pre_process_data/notebooks/0.fix_pathing.ipynb:1]: Note to make sure this is nbconverted.

File: 4.process_features/notebooks/1.annotate_sc.ipynb
Comment (jenna-tomkinson) [inline 4.process_features/notebooks/1.annotate_sc.ipynb:1]: Might consider adding these pycytominer functions into one notebook to reduce the number of files and simplify.

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I made a few comments for you to address, but after that feel free to merge!

File: 1.pre_process_data/notebooks/1.generate_platemap.ipynb
Comment (MikeLippincott) [reply 1.pre_process_data/notebooks/1.generate_platemap.ipynb:1]: converted

File: 1.pre_process_data/notebooks/0.fix_pathing.ipynb
Comment (MikeLippincott) [reply 1.pre_process_data/notebooks/0.fix_pathing.ipynb:1]: converted

File: 4.process_features/notebooks/1.annotate_sc.ipynb
Comment (MikeLippincott) [reply 4.process_features/notebooks/1.annotate_sc.ipynb:1]: Out of part laziness and part wanting to compartmentalize each process, I am going to keep it how it is

**Pull request merged**